static/src/javascripts/projects/common/modules/onward/breaking-news.js

```diff
@@ -88,11 +89,13 @@ define([
                 if (alerts.length) {
                     $breakingNews = $breakingNews || bonzo(qwery('.js-breaking-news-placeholder'));
                     marque36icon = svgs('marque36icon');
+                    closeIcon = svgs('closeCentral');
```

Comment:
this ought to be `closeCentralIcon` really, so you know it comes with the `icon` class